xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/VideoFilterShader.h

```diff
@@ -54,7 +54,7 @@ namespace Shaders {
     // shader attribute handles
     GLint m_hSourceTex;
     GLint m_hStepXY;
-    GLint m_hStretch;
+    GLint m_hStretch = 0;
```

Comment:
the other one is initialized here: https://github.com/xbmc/xbmc/blob/8405bbf242a73e7b88641c7db4a9808fd46a2f47/xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/VideoFilterShader.cpp#L51